.github/dead_link_check_config.json

```diff
@@ -33,6 +36,7 @@
   "fallbackRetryDelay": "1s",
   "aliveStatusCodes": [
     200,
-    401
+    401,
+    403
```

Comment:
How about we only add the cloudflare link into the ignorePatterns, instead of ignore 403?
```json
{
  "ignorePatterns": [
    {
      "pattern": "^https://techhenzy.com/"
    },
```

Reply:
ok

Reply:
Now the 403 appears again 😭 
Sorry for misleading.
Let's ignore 403 ....